example/settings/base.py

```diff
@@ -122,6 +128,34 @@
     'apps.test_security',
 )
 
+
+def _json_serializer(data, **kwargs):
+    serialized_data = {}
+    for k, v in data.items():
+        if isinstance(v, Enum):
+            serialized_data[k] = v.name
+        else:
+            serialized_data[k] = v
+    return json.dumps(serialized_data, cls=DjangoJSONEncoder)
+
+
+structlog.configure(
+    processors=[
+        structlog.processors.EventRenamer("message"),
+        structlog.stdlib.filter_by_level,
+        structlog.stdlib.add_log_level,
+        structlog.stdlib.add_logger_name,
+        structlog.processors.TimeStamper(fmt="iso"),
+        structlog.stdlib.PositionalArgumentsFormatter(),
+        structlog.processors.JSONRenderer(_json_serializer),
+    ],
+    context_class=dict,
+    logger_factory=structlog.stdlib.LoggerFactory(),
+    wrapper_class=structlog.stdlib.BoundLogger,
+    cache_logger_on_first_use=True,
+)
```

Comment:
This is an example configuration needed to enable logging with structlog. 

As the serialization is left up to the library user it gives the user more freedom to choose the logging formatting.